modern-cpp-font-lock.el

```diff
@@ -147,27 +166,30 @@ http://en.cppreference.com/w/cpp/language/integer_literal"
         (,literal-dec-regexp (1 font-lock-constant-face)
                              (2 font-lock-keyword-face))))))
 
-(defvar modern-c++-font-lock-keywords
+(defvar modern-c++-font-lock-keywords nil)
+
```

Comment:
If `defvar` is used, it only runs once. Customization from user may not be used. `defun` generates a the keywords upon calls.